packages/editor/editor.css

```diff
@@ -23,33 +23,39 @@
 	--radius-4: 11px;
 
 	/* Canvas z-index */
-	--layer-cvs-hidden: -999999;
-	--layer-cvs-background: 100;
-	--layer-cvs-grid: 150;
-	--layer-cvs-shapes: 300;
-	--layer-cvs-overlays: 400;
-	--layer-cvs-blocker: 10000;
+	--layer-canvas-hidden: -999999;
```

Comment:
too hard to read - sounds like CVS and Ols - my brain doesn't know what these are :P